core-bundles/language/org.eclipse.vorto.editor.datatype/src/org/eclipse/vorto/editor/datatype/Datatype.xtext

```diff
@@ -27,12 +28,16 @@ QualifiedName:
 	ID ('.' ID)*
 ;
 
+VortolangVersion:
+  '1.0'
+;
+
 Type:
 	Entity | Enum
 ;
 
 Entity:
-   'vortolang' lang=VortoLangVersion
+   'vortolang' lang=FLOAT
```

Comment:
Why is the vortolang definition in the entity different from the one in the enum (line 56)?